src/legacy/core_plugins/vis_default_editor/public/components/controls/raw_json.tsx

```diff
@@ -34,48 +32,69 @@ function RawJsonParamEditor({
   setValue,
   setTouched,
 }: AggParamEditorProps<string>) {
+  const [isFormValid, setFormValidity] = useState(true);
+  const [editorReady, setEditorReady] = useState(false);
+
+  const editorTooltipText = i18n.translate('visDefaultEditor.controls.jsonInputTooltip', {
+    defaultMessage:
+      "Any JSON formatted properties you add here will be merged with the elasticsearch aggregation definition for this section. For example 'shard_size' on a terms aggregation.",
+  });
+
+  const jsonEditorLabelText = i18n.translate('visDefaultEditor.controls.jsonInputLabel', {
+    defaultMessage: 'JSON input',
+  });
+
   const label = (
     <>
-      <FormattedMessage id="visDefaultEditor.controls.jsonInputLabel" defaultMessage="JSON input" />{' '}
-      <EuiIconTip
-        position="right"
-        content={i18n.translate('visDefaultEditor.controls.jsonInputTooltip', {
-          defaultMessage:
-            "Any JSON formatted properties you add here will be merged with the elasticsearch aggregation definition for this section. For example 'shard_size' on a terms aggregation.",
-        })}
-        type="questionInCircle"
-      />
+      {jsonEditorLabelText}{' '}
+      <EuiIconTip position="right" content={editorTooltipText} type="questionInCircle" />
     </>
   );
-  const isValid = isValidJson(value);
 
-  const onChange = (ev: React.ChangeEvent<HTMLTextAreaElement>) => {
-    const textValue = ev.target.value;
-    setValue(textValue);
-    setValidity(isValidJson(textValue));
+  const onChange = (newValue: string) => {
+    setValue(newValue);
   };
 
-  useEffect(() => {
-    setValidity(isValid);
-  }, [isValid]);
+  const onEditorValidate = (annotations: unknown[]) => {
+    // The first onValidate returned from EuiCodeEditor is a false positive
```

Comment:
Could you give me a case when `annotations` is a false positive at the first call?

Reply:
Thanks for explanation.